web/app/scripts/filterbar/filterbar.html

```diff
@@ -5,6 +5,8 @@
         </ul>
         <ul class="nav navbar-nav">
             <li date-range-field="__dateRange" class="dropdown"></li>
+        </ul>
+        <ul class="nav navbar-nav">
```

Comment:
Apparently the two date filters were excessively close to each other because they shared the same `<ul>` element... normally each filter has its own `<ul>` with exactly one `<li>` inside.

May be worth at some point investigating the styles and if it's feasible to have a single `<ul>` with each filter represented by a `<li>` inside that shared `<ul>`.

Reply:
Good thought; we used to have a task to address the filter bar styling but it got removed in the latest project iteration. However, I've added this note to the epic where it used to be and if we have extra time at the end of the project we can address it.